modules/social_features/social_follow_taxonomy/modules/social_follow_tag/social_follow_tag.install

```diff
@@ -160,14 +160,19 @@ function _social_follow_tag_update_existing_messages(array &$sandbox): void {
     // Get all message IDs than should be updated.
     $query = $message_storage->getQuery()
       ->condition('template', 'create_node_following_tag_stream');
-    $mids = $query->execute();
+    $mids = $query
+      ->accessCheck(TRUE)
```

Comment:
We can put this accessCheck to false, as when run it will be anonymous and only takes what the anonymous user can view. In this case we want to update this message template regardless to if the current user has access to it.